### DISCUSSION TITLE
Log entry [Telemetry]: Activity <nw_activity ...

### DISCUSSION THREAD

**Discussion by kmatzka** [TelemetryDeck/Viewer#319] [Q&A]
During my first trials with TelemetryDeck today I have found two log entries like the following two lines repeatedly:

> [Telemetry]: Activity <nw_activity 12:2[80E7BD37-D3DB-4648-B606-2F5C52FB342F] (reporting strategy default)> on Task <2DA76592-DDEE-475F-9FE1-4816D664B105>.<2> was not selected for reporting
> 
> Task <2DA76592-DDEE-475F-9FE1-4816D664B105>.<2> request https://nom.telemetrydeck.com/api/v1/apps/MY_APP_ID/signals/multiple/ is NOT allowed to set HSTS for main doc (null)

Did not show when my app has been running on an iOS 16.7 iPhone.
Shows up on an iOS 17.2 iPad.
Both devices in DEBUG mode.

What do those entries mean?

**Comment by winsmith** [comment]
To be honest I've never seen these before! HSTS is a reference to the [HTTP Strict Transport Security Header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security), maybe iOS 17 is hinting at more stringent security policies in the future here. I'll keep an eye out.
